deploy/bin/worker_type_utils.js

```diff
@@ -0,0 +1,69 @@
+const tc = require('taskcluster-client');
+const promiseRetry = require('promise-retry');
```

Comment:
I'm not sure the use of this library here.  The taskcluster client already has retries in it.

Reply:
When using [manage.js](https://github.com/taskcluster/aws-provisioner/blob/master/src/manage.js) I had some issues.

Reply:
do you remember the issues? These sound like something we need to open up against aws-provisioner then.

Reply:
When doing batch worker-type updates, sometimes one randomly failed. Adding `promiseRetry` fixed it.